.github/workflows/ruby_test.yml

```diff
@@ -87,8 +87,87 @@ jobs:
     # Sets up the matrix for knapsack builds
     strategy:
       matrix:
-        ci_node_total: [3]
-        ci_node_index: [0, 1, 2]
+        ci_node_total: [2]
+        ci_node_index: [0, 1]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true # runs 'bundle install' and caches installed gems automatically
+      - name: Setup node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+      # Establish a cache of js modules to improve performance
+      - name: Cache js
+        uses: actions/cache@v4
+        with:
+          path: |
+            public/assets
+            node_modules
+          key: ${{ runner.os }}-tests-${{ hashFiles('yarn.lock') }}
+          # If we don't find the specific cache we want, fallback to the last raketests
+          # cache, then finally any cache for this repo.
+          # Github looks for the newest cache beginning with the first entry, before
+          # falling back the the second if none is present.
+          restore-keys: |
+            ${{ runner.os }}-tests-
+            ${{ runner.os }}-
+      # Install only the gems needed for testing
+      # Keep an eye on https://github.com/rubygems/bundler-features/issues/59
+      # in case bundler add an only flag
+      # We also set the install path to vendor/bundle to assist with out caching
+      # bin/setup handles the rest of the configuration for us.
+      - name: Setup environment
+        env:
+          DBPORT: ${{ job.services.mysql.ports[3306] }}
+        run: |
+          bin/setup
+          bundle exec rake assets:precompile
+      # Actually run our tests
+      - name: Run rspec tests
+        env:
+          KNAPSACK_PRO_CI_NODE_TOTAL: ${{ matrix.ci_node_total }}
+          KNAPSACK_PRO_CI_NODE_INDEX: ${{ matrix.ci_node_index }}
+          KNAPSACK_PRO_TEST_FILE_EXCLUDE_PATTERN: "spec/features/**{,/*/**}/*_spec.rb"
+          DBPORT: ${{ job.services.mysql.ports[3306] }}
+        run: bin/knapsack_pro_rspec
```

Comment:
is it worth creating a new run command to separate features from unit tests instead of using file inclusion/exclusion?

Reply:
Yeah will that break knapsack if we split it up? Would be useful for local too just thinking if file inclusion/exclusion is the cleanest way to split them.